install.sh

```diff
@@ -10,7 +10,7 @@ php -r "unlink('composer-setup.php');"
 sudo apt-get install nodejs npm
 
 # install npm gulp cli globally for css minification
-npm install -g gulp-cli
+sudo npm install -g gulp-cli
```

Comment:
Instead of using sudo here, can we call this out, whosoever runs this script should run it using sudo. 
FYI : https://askubuntu.com/questions/425754/how-do-i-run-sudo-command-inside-a-script/425990
Is this script meant to be run manually ? or it is called by some function ?